### PR TITLE
Allow name edition in CE

### PR DESCRIPTION
**Pull request by soupette**
Signed-off-by: soupette <cyril.lpz@gmail.com>

#### Description of what you did:
Allow role name edition for CE.

### CODE REVIEW

File: packages/strapi-admin/ee/controllers/role.js [hunk 1]
```diff
@@ -3,11 +3,11 @@
 const { yup, formatYupErrors } = require('strapi-utils');
 const {
   validateRoleCreateInput,
-  validateRoleUpdateInput,
-  validateRolesDeleteInput,
   validateRoleDeleteInput,
+  validateRolesDeleteInput,
 } = require('../validation/role');
 const { validatedUpdatePermissionsInput } = require('../validation/permission');
+const { validateRoleUpdateInput } = require('../../validation/role');
```
Comment (petersg83) [inline packages/strapi-admin/ee/controllers/role.js:10]: You can remove this line and remove the whole function at line 35. It is the same as in CE

**Comment by petersg83** [comment]
LGTM :)
Please wait for the review of @alexandrebodin I think he mights have an opinion on "develop:ce"

File: examples/getstarted/package.json [hunk 2]
```diff
@@ -5,6 +5,7 @@
   "description": "A Strapi application.",
   "scripts": {
     "develop": "strapi develop",
+    "develop:ce": "STRAPI_DISABLE_EE=true strapi develop",
```
Comment (alexandrebodin) [inline examples/getstarted/package.json:8]: you can also add one for the build step then

**Comment by alexandrebodin** [comment]
I think we need to do sth in case we rename the Super Admin role, the error messages that say you need to have at least one super admin won't make much sense.

**Comment by JabStrapi** [comment]
You are right @alexandrebodin, in my opinion and for consistency matters, we should prevent only the Super Admin renaming from the Admin Panel (it would still be possible in DB).

**Comment by derrickmehaffy** [comment]
I agree @JabStrapi

**Pull request merged**